### ISSUE TITLE
[a11y] Audit Restitution

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1145]
# Vérification de l'accessibilité du composant : Restitution

Afin d'évaluer le niveau d'accessibilité d'un composant, il faut effectuer les actions ci-dessous.

## Analyse statique d'accessibilité

Remplir les champs ci-dessous, en se basant sur l'onglet accessibilité de chaque composant dans le storybook.

![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/74aa4e24-2055-4a58-9b20-4a9249630e39)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 10

En plus mettre une capture de l'onglet en commentaire de cette issue.

## Analyse manuelle

- [x] Est ce que le composant est navigable au clavier (tab) ?
Composant d'affichage, le bouton Modifier est en bien navigable au clavier.
- [x] Est ce que le composant est navigable avec NVDA ?
- [ ] Est ce que le composant est navigable avec un autre lecteur d'écran ?

## Vérification sémantique

- [x] Vérifier l'implémentation sémantique des éléments html ?
Utiliser des éléments heading pour les titres
- [x] Vérifier la validité HTML W3C (https://validator.w3.org/#validate_by_input) ?
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/e323ca62-e9e2-468d-afae-2f77e36a315b)
L'article et la section de contenu ne contiennent pas de titre. Dans le header, le titre est dans une div plutôt qu'un élément de heading. Pareil pour les titres au sein du contenu de la restitution.
- [x] Vérifier la validité CSS (https://jigsaw.w3.org/css-validator/#validate_by_input) ?

## Problèmes détectés

Si vous avez détectés un défaut d'accessibilité, veuillez créer une issue décrivant le défaut et la rattacher à cette issue.

**Comment by MartinWeb** [comment]
Remontées à corriger :

- Utiliser des heading pour le titre du header et les titres du contenu

**Comment by MartinWeb** [comment]
Report sur le projet design system
